show-location-history/src/main/java/com/esri/arcgismaps/sample/showlocationhistory/MainActivity.kt

```diff
@@ -0,0 +1,148 @@
+import com.arcgismaps.geometry.Geometry
+import com.arcgismaps.geometry.Point
+import com.arcgismaps.geometry.Polyline
+import com.arcgismaps.geometry.PolylineBuilder
+import com.arcgismaps.geometry.SpatialReference
+import com.arcgismaps.location.LocationDisplayAutoPanMode
+import com.arcgismaps.location.SimulatedLocationDataSource
+import com.arcgismaps.location.SimulationParameters
+import com.arcgismaps.mapping.ArcGISMap
+import com.arcgismaps.mapping.BasemapStyle
+import com.arcgismaps.mapping.Viewpoint
+import com.arcgismaps.mapping.symbology.SimpleLineSymbol
+import com.arcgismaps.mapping.symbology.SimpleLineSymbolStyle
+import com.arcgismaps.mapping.symbology.SimpleMarkerSymbol
+import com.arcgismaps.mapping.symbology.SimpleMarkerSymbolStyle
+import com.arcgismaps.mapping.symbology.SimpleRenderer
+import com.arcgismaps.mapping.view.Graphic
+import com.arcgismaps.mapping.view.GraphicsOverlay
+import com.esri.arcgismaps.sample.showlocationhistory.databinding.ActivityMainBinding
+import com.google.android.material.snackbar.Snackbar
+import kotlinx.coroutines.launch
+import kotlinx.datetime.Clock
+
+class MainActivity : AppCompatActivity() {
+
+    private var isTrackLocation: Boolean = false
+
+    // set up data binding for the activity
+    private val activityMainBinding: ActivityMainBinding by lazy {
+        DataBindingUtil.setContentView(this, R.layout.activity_main)
+    }
+
+    private val mapView by lazy {
+        activityMainBinding.mapView
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        // authentication with an API key or named user is
+        // required to access basemaps and other location services
+        ArcGISEnvironment.apiKey = ApiKey.create(BuildConfig.API_KEY)
+        lifecycle.addObserver(mapView)
+
+        // create a center point for the data in West Los Angeles, California
+        val center = Point(-13185535.98, 4037766.28, SpatialReference(102100))
+
+        // create a graphics overlay for the points and use a red circle for the symbols
+        val locationHistoryOverlay = GraphicsOverlay()
+        val locationSymbol = SimpleMarkerSymbol(SimpleMarkerSymbolStyle.Circle, Color.red, 10f)
+        locationHistoryOverlay.renderer = SimpleRenderer(locationSymbol)
+
+        // create a graphics overlay for the lines connecting the points and use a blue line for the symbol
+        val locationHistoryLineOverlay = GraphicsOverlay()
+        val locationLineSymbol = SimpleLineSymbol(SimpleLineSymbolStyle.Solid, Color.green, 2.0f)
+        locationHistoryLineOverlay.renderer = SimpleRenderer(locationLineSymbol)
+
+        mapView.apply {
+            // create and add a map with a navigation night basemap style
+            map = ArcGISMap(BasemapStyle.ArcGISNavigationNight)
+            setViewpoint(Viewpoint(center, 7000.0))
+            graphicsOverlays.addAll(listOf(locationHistoryOverlay, locationHistoryLineOverlay))
```

Comment:
please double-check, by design,  `locationHistoryLineOverlay ` should be under `locationHistoryLineOverlay `,  the order needs to reverse,  it should be
```
 graphicsOverlays.addAll(listOf(locationHistoryLineOverlay, locationHistoryOverlay))
```

Reply:
@changanxian I am not sure, I understand your comment. both are of type graphicsOverlay() and added to the list of GO's. How does the order you mention matter in this case?